packages/core/upload/admin/src/translations/en.json

```diff
@@ -115,14 +115,15 @@
   "sort.name_desc": "Reverse alphabetical order (Z to A)",
   "sort.updated_at_asc": "Oldest updates",
   "sort.updated_at_desc": "Most recent updates",
-  "list-table-header-actions": "actions",
-  "list-table-header-preview": "preview",
-  "list-table-header-name": "name",
-  "list-table-header-ext": "extension",
-  "list-table-header-size": "size",
-  "list-table-header-createdAt": "created",
-  "list-table-header-updatedAt": "last update",
-  "list-table-header-sort": "Sort on {label}",
+  "list.table.header.actions": "actions",
```

Comment:
Oh thanks. I thought the same but didn't want to start bikeshedding :D

Reply:
I don't understand why I used this syntax, triggered me when I saw it 😂